.env

```diff
@@ -85,7 +85,7 @@ _APP_MAINTENANCE_DELAY=
 _APP_MAINTENANCE_RETENTION_CACHE=2592000
 _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
-_APP_MAINTENANCE_RETENTION_AUDIT=1209600
+_APP_MAINTENANCE_RETENTION_AUDIT=project=1209600,console=15778800
```

Comment:
Let's still use separate env vars and just fetch them both in the `auditRetention` resource